.travis.yml

```diff
@@ -1,7 +1,18 @@
 language: python
-python:
-    - '3.6'
-    - '3.7'
+matrix:
+  include:
+    - python: "3.6"
+    - python: "3.7"
+    - python: "3.8"
+    - python: "3.8-dev"
+    - python: "3.9-dev"
+    - python: "pypy3"
+      env: NO_MYPY=true
```

Comment:
```suggestion
```
You don't need this environment variable. In the example I linked, this is used to disable mypy type checking under PyPy since it isn't supported.